fastly.yaml

```diff
@@ -0,0 +1,4 @@
+---
+vhost:
```

Comment:
It isn't obvious to me that this (and correct me if I'm wrong here) is actually a configurable key. I suggest that Yaml like this might be a better example:

```yaml
---
frontend_webservers:
  production:
    service_id: a1b2c3d4e5f6g7
  staging:
    service_id: a2b3c4d5e6f7g8
backend_api_requests:
  production:
    service_id: a3b4c5d6e7f8g9
```

I'd also drop the term `vhost` and refer instead to services, pools, environments or something else everywhere it's used.